File: algorithm/src/main/java/baekjoon/p02577/Main.java
package baekjoon.p02577;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        for (int solve : solve(sc.nextInt(), sc.nextInt(), sc.nextInt())) {
            System.out.println(solve);
        }
    }

    private static int[] solve(int a, int b, int c) {
        int[] ret = new int[10];
        int mul = (a * b * c);

        while (mul != 0) {
            ret[mul % 10]++;
            mul /= 10;
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p02577/Main.kt
package baekjoon.p02577

fun main(args: Array<String>) {
    val a = readLine()!!.toInt()
    val b = readLine()!!.toInt()
    val c = readLine()!!.toInt()

    for (solve in solve(a, b, c)) {
        println(solve)
    }
}

private fun solve(a: Int, b: Int, c: Int): IntArray {
    val ret = IntArray(10)
    var mul = (a * b * c)

    while (mul != 0) {
        ret[mul % 10]++
        mul /= 10
    }

    return ret
}